spec/support/fake_nation_builder.rb

```diff
@@ -3,7 +3,7 @@
 
 class FakeNationBuilder < Sinatra::Base
 
-  get '/api/v1/lists/' do
+  get '/supporter_counts_for_website' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.